tinygrad/tensor.py

```diff
@@ -303,6 +303,32 @@ def chunk(self, num, dim):
       slice_params[i][dim] = (k, min(self.shape[dim], k+self.shape[dim]//num))
     return [self.slice(p) for p in slice_params]
 
+  @staticmethod
```

Comment:
Why are these staticmethod?

Reply:
That's not a good reason, why have the extra `@staticmethod` line and get less functionality.

Reply:
changed